recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "xcube-sh" %}
-{% set version = "0.5.0" %}
+{% set version = "0.6.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/dcs4cop/xcube-sh/archive/{{ version }}.tar.gz
```

Comment:
This download string fails in the tests. It expects 0.6.0.tar.gz but the repo provides v0.6.0.tar.gz.